velox/vector/arrow/Bridge.h

```diff
@@ -36,6 +36,7 @@ struct ArrowOptions {
   bool flattenDictionary{false};
   bool flattenConstant{false};
   TimestampUnit timestampUnit = TimestampUnit::kNano;
+  bool exportToView = false;
```

Comment:
does this control only if we export from velox string view into arrow string view? or also the import? Can we make it more explicit, like `exportToStringView`?

Reply:
It only controls the export from velox string view to arrow string view. Updating the name to exportToStringView